crates/sui-graphql-rpc/src/types/error.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use async_graphql::{Error, ErrorExtensions};
+
+#[derive(Debug, thiserror::Error)]
```

Comment:
I saw in #13378, a new file of src/error.rs was added, do we need both of them?

Reply:
Yes, we should merge these two files -- @wlmyng , could you rebase on top of my PR?